roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -6,11 +6,21 @@
        force=yes
 
 - name: Copy NGI-RNAseq config
-  template: src="ngi-rnaseq.config.j2" dest="{{ ngi_pipeline_conf }}/ngi-rnaseq_{{ item.site }}.config"
+  template: src="{{ ngi_rnaseq_dest }}/conf/uppmax.config" dest="{{ ngi_pipeline_conf }}/ngi-rnaseq_{{ item.site }}.config"
   with_items:
   - { site: "sthlm", proj: "{{ ngi_pipeline_sthlm_delivery }}" }
   - { site: "upps", proj: "{{ ngi_pipeline_upps_delivery }}" }
 
+- name: Apply NGI specific config changes
+  lineinfile: dest="{{ ngi_pipeline_conf }}/ngi-rnaseq_{{ item.site }}.config"
```

Comment:
Have you tested this, or is this as expected? Because `lineinfile` will here just ensure that the lines that you specify will be added, and when I looked at https://github.com/SciLifeLab/NGI-RNAseq/blob/master/conf/uppmax.config I was led to believe that the config should have different syntax. But perhaps that is how Nextflow works?